src/components/UNSTABLE_Pagination/_mixins.scss

```diff
@@ -0,0 +1,148 @@
+////
+/// Unstable pagination mixins.
+/// @group unstable-pagination
+/// @copyright IBM Security 2019
+////
+
+@import '../../globals/index';
```

Comment:
Instead of importing the `../../globals/index` file, can we import only the required modules? This follows Carbon's approach and will help us reduce the Sass compilation time in the future.

You can leverage the `modular-styles` branch as a reference - https://github.com/carbon-design-system/ibm-security/tree/44/modular-styles

Reply:
Would it be worth reusing Carbon's pagination styles here where we can (if possible) to avoid duplication and keeping it in sync with the original implementation?

Reply:
I'm happy to only import required modules! But I'd rather not import the Carbon pagination styles as-is. This component is being refactored so I think it is reasonable to also re-examine the styles.

Reply:
That's fair, as long they're not deviating too much from Carbon's original styles - I'm hoping this is something that can be swiftly contributed back to reduce our need for maintaining it. 🎉